### PR TITLE
Feature bot 1437 add language in user edition page

### PR DESCRIPTION
**Pull request by kun2985**
# Description

This PRadd language in user edition page

## Issue (optional)

Issue link: bot-1437

Blocked by https://github.com/CanalTP/NmmPortalBundle/pull/65

### CODE REVIEW

**Comment by pthegner** [comment]
retest this please

**Comment by pthegner** [comment]
retest this please

File: Controller/UserController.php [hunk 1]
```diff
@@ -178,18 +177,27 @@ public function editProfilProcessForm($user)
      */
     public function editProfilAction()
     {
+        //to translate the flashbag to chosen language
+        $request = $this->getRequest();
```
Comment (Netmisa) [inline Controller/UserController.php]: You can get Request from argument here

File: Controller/UserController.php [hunk 2]
```diff
@@ -178,18 +177,27 @@ public function editProfilProcessForm($user)
      */
     public function editProfilAction()
```
Comment (Netmisa) [inline Controller/UserController.php]:
```suggestion
    public function editProfilAction(Request $request)
```

File: Controller/UserController.php [hunk 3]
```diff
@@ -178,18 +177,27 @@ public function editProfilProcessForm($user)
...
     {
+        //to translate the flashbag to chosen language
+        $request = $this->getRequest();
+        $postData = $request->request->get('edit_user_profil');
```
Comment (Netmisa) [inline Controller/UserController.php]: Why you doesn't use $form to get value from  `edit_user_profil_language` field ?

Comment (kun2985) [reply Controller/UserController.php]: It's for creating form with the selected language. If not, the sucess message will be the old language.

File: Controller/UserController.php [hunk 4]
```diff
@@ -176,20 +175,28 @@ public function editProfilProcessForm($user)
...
      * Displays a form to edit profil of current user.
      */
-    public function editProfilAction()
+    public function editProfilAction(Request $request)
     {
+        //to translate the flashbag to chosen language
+        $postData = $request->request->get('edit_user_profil');
+        if ($postData && $request->getLocale() !== $postData['language']) {
+            $request->setLocale($postData['language']);
+            $this->editProfilAction();
```
Comment (Netmisa) [inline Controller/UserController.php]: `$request` variable was forgotten

Comment (Netmisa) [inline Controller/UserController.php]:
```suggestion
            $this->editProfilAction($request);
```

**Pull request merged**